src/main/java/de/bastisdevelopment/mykitaapp/dtos/KitaDTO.java

```diff
@@ -0,0 +1,35 @@
+package de.bastisdevelopment.mykitaapp.dtos;
+
+import de.bastisdevelopment.mykitaapp.items.AppUser;
+import de.bastisdevelopment.mykitaapp.items.KitaDBItem;
+import lombok.Getter;
+
+@Getter
```

Comment:
Why only @Getter and not @Data? You propably need an empty constructor for spring
```suggestion
@Data
@NoArgsConstructor
```